tests/test_lint_recipe.py

```diff
@@ -336,7 +612,7 @@ def test_bad_about_license(self):
                 "license": "unknown",
             }
         }
-        lints, hints = linter.lintify_meta_yaml(meta)
```

Comment:
We cannot push api changes like this without a major version bump and a lot of discussion.  further, if any api changes were pushed in the previous PRs, they need to be backed out.

cc @wolfv

Reply:
That seems like a internal API, correct?
I haven't found any usage outside of conda-smithy with a Github code search. 

We'll double check to not break any other project import `conda-smithy`.

Reply:
Internal APIs begin with an underscore in Python. Any other API is always external and cannot be changed without a major version change and discussion. We cannot change them.

cc @conda-forge/core